__tests__/app.test.js

```diff
@@ -86,6 +86,46 @@ describe("GET /api/articles/:article_id", () => {
   });
 });
 
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: an array of comments for the given article_id with the correct properties", () => {
+    return request(app)
+      .get("/api/articles/3/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toBeSortedBy("created_at", { descending: true });
```

Comment:
please add a length check as this will pass for an empty array, when in some cases it shouldn't!